.github/configuration/upgrade-test-configuration.yml

```diff
@@ -7,4 +49,18 @@ upgrade-version: [[
     version: target.latest,
     upgrade-type: major
   }
+],
+[
+  {
+    version: 14.5,
+    upgrade-type: null
+  },
+  { 
+    version: 14.6,
+    upgrade-type: minor 
+  },
+  { 
+    version: target.latest,
+    upgrade-type: major 
+  }
```

Comment:
Let's also add direct upgrade path from 14.6 to latest

Reply:
Done